src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -1407,18 +1412,12 @@ public static void Main()
 }
 ";
         var vbCompilation = CreateVisualBasicCompilation(vbSource);
-        var csCompilation = CreateCompilation(csSource, parseOptions: TestOptions.RegularWithListPatterns, references: new[] { vbCompilation.EmitToImageReference() });
-        // PROTOTYPE(list-patterns) Unsupported because the lookup fails not that the indexer is static
+        var csCompilation = CreateCompilation(csSource, references: new[] { vbCompilation.EmitToImageReference() });
+        // Note: the VB indexer's name is "Item" but C# looks for "this[]", but we can't put Default on Shared indexer declaration
```

Comment:
>but we can't put Default on Shared indexer declaration [](http://example.com/codeflow?start=76&length=54)

Can we do that in IL? Is it worth testing a property like that?

Reply:
We have a test with a static indexer from IL, see `SlicePattern_LengthAndIndexAndSliceAreStatic`